libs/stache/src/lib/modules/wrapper/title.service.ts

```diff
@@ -1,23 +1,29 @@
-import { Injectable } from '@angular/core';
+import { Injectable, inject } from '@angular/core';
 import { Title } from '@angular/platform-browser';
 import { SkyAppConfig } from '@skyux/config';
 
 @Injectable()
 export class StacheTitleService {
-  constructor(
-    private title: Title,
-    private configService: SkyAppConfig,
-  ) {}
+  readonly #title = inject(Title);
+  readonly #appConfig = inject(SkyAppConfig);
 
-  public setTitle(...parts: string[]) {
-    let windowTitle = this.configService.skyux.app.title;
+  public setTitle(...parts: string[]): void {
+    const appTitle = this.#appConfig.skyux.app?.title;
```

Comment:
Without the null check, the `app.title` throws a confusing error if the consumer does not set`app.title` in their `skyuxconfig.json` file. This change allows for that value to be undefined.